package.json

```diff
@@ -204,5 +204,10 @@
   },
   "optionalDependencies": {
     "cypress": "9.7.0"
+  },
+  "overrides": {
```

Comment:
This is overriding a sub dependency? What's the reasoning?

Reply:
Yes, it updates the sub dependency. The latest crypto-browserify still has pbkdf2 3.1.2 which has critical audit issue

Reply:
Fair. Is there a ticket to remove this override when (presumably) crypto-browserify updates?

Reply:
[CUMULUS-4171](https://bugs.earthdata.nasa.gov/browse/CUMULUS-4171)